.vortex/tooling/src/helpers.php

```
@@ -0,0 +1,707 @@
+ *   - status: HTTP status code
+ *   - body: Response body (empty string when save_to is used)
+ *   - error: Error message if any
+ *   - info: Request info array
+ */
+function request(string $url, array $options = []): array {
+  // @codeCoverageIgnoreStart
+  if (!function_exists('curl_init')) {
+    fail('curl extension is not available.');
+  }
+  // @codeCoverageIgnoreEnd
+  $ch = curl_init($url);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $ch. Configured minimum length is 3. (undefined)

(ShortVariable)